lymph/node.py

```diff
@@ -26,19 +27,19 @@ def __init__(
         is provided.
         """
         self.name = name
-        self.state = state
 
         if allowed_states is None:
             allowed_states = [0, 1]
 
         _allowed_states = []
-        for state in allowed_states:
+        for state1 in allowed_states:

```

Comment:
Don't enumerate variable names. Either use a more descriptive name, like `allowed_state`, or make it clear that it is only temporarily used inside the loop, e.g. by naming it `_state` or even just `s`.